rest-api/grant_permissions.sql

```diff
@@ -0,0 +1,7 @@
+GRANT SELECT ON ${db_name}.* TO '${READONLY_DB_USER}'@'%';
+GRANT SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES ON ${db_name}.*
+TO '${RDR_DB_USER}'@'%';
```

Comment:
If you're going to break the lines up, please indent.

Reply:
fixed